Update editor workflow to reflect read-only Airtable

noamross opened the pull request:
Part of #688

We previously requested editors keep notes on reviewer outreach in the AirTable, but data stability, privacy and price-related reasons this database is no longer manually editable.  This PR updates editor workflow and guidance to reflect this.

yabellini commented on softwarereview_editor.es.Rmd:
```diff
@@ -70,28 +70,17 @@ Cuando estas fuentes de información no sean suficientes,
 - Puedes publicar la búsqueda en las redes sociales.
 
-##### Consejos para la búsqueda en Airtable {#tips-for-reviewer-search-in-airtable}*Airtable*
+- Puedes tuitear sobre la búsqueda de revisores.
```
```suggestion
- Puedes publicar la búsqueda en las redes sociales.
```

yabellini replied:
I suggest we also change this in the English version from
- You might tweet about the reviewer search.
to
- You might post on social media about the reviewer search.

yabellini commented on softwarereview_editor.es.Rmd:
```diff
@@ -70,28 +70,17 @@ Cuando estas fuentes de información no sean suficientes,
+##### Consejos para la búsqueda de revisores en Airtable {#tips-for-reviewer-search-in-airtable}
 
-- filtros (por favor, elimínalos una vez que hayas terminado), ver ejemplo más abajo.
+Puedes utilizar los filtros, la clasificación y la búsqueda para identificar a los revisores con una experiencia concreta:
```
```suggestion
Puedes utilizar filtros, clasificación y búsqueda para identificar personas para la revisión con una experiencia concreta:
```

noamross commented:
I added my first attempt at using {babeldown} to generate a translation.  I've attempted to only stage the section of the file that I changed.  Babeldown translates the whole file and many sections are different when it does so. (Likely a change anywhere has implications everywhere in the model, and also Babeldown's model may change over time.)

yabellini commented on softwarereview_editor.es.Rmd:
```diff
@@ -70,28 +70,17 @@ Cuando estas fuentes de información no sean suficientes,
+Puedes utilizar los filtros, la clasificación y la búsqueda para identificar a los revisores con una experiencia concreta:
 
-![Captura de pantalla de la interfaz de filtros de Airtable con un filtro sobre conocimientos técnicos que tiene que incluir química y en áreas técnicas tienen que incluir integración continua (interfaz y etiquetas en inglés)](images/airtable.png)
+![Captura de pantalla de la interfaz de filtros de Airtable con un filtro sobre experiencia en dominios que tienen que incluir la química y áreas técnicas que tienen que incluir la integración continua](images/airtable.png)
```
```suggestion
![Captura de pantalla de la interfaz de filtros de Airtable con un filtro sobre experiencia en disciplinas que debe incluir química y en conocimientos técnicos que tienen que incluir integración continua](images/airtable.png)
```

yabellini commented on softwarereview_editor.es.Rmd:
```diff
@@ -70,28 +70,17 @@ Cuando estas fuentes de información no sean suficientes,
-No introduzcas más información que su nombre de usuario de GitHub, su nombre y su dirección de correo electrónico.
-Pide a quienes que podrían potencialmente revisar que completen el [formulario de Airtable](https://ropensci.org/software-reviewer).
+Comprueba la opinión más reciente del revisor y evita a cualquiera que haya revisado a alguien en los últimos seis meses.
+Asimismo, comprueba si un revisor primerizo ha indicado que `require_mentorship`.
```
```suggestion
Asimismo, comprueba si una persona que es nueva revisando ha indicado que `require_mentorship`.
```

yabellini commented on softwarereview_editor.es.Rmd:
```diff
@@ -70,28 +70,17 @@ Cuando estas fuentes de información no sean suficientes,
-Sólo añade personas a Airtable si aceptan revisar (de lo contrario, las personas deben ofrecerse voluntariamente rellenando el formulario de Airtable).
-No introduzcas más información que su nombre de usuario de GitHub, su nombre y su dirección de correo electrónico.
-Pide a quienes que podrían potencialmente revisar que completen el [formulario de Airtable](https://ropensci.org/software-reviewer).
+Comprueba la opinión más reciente del revisor y evita a cualquiera que haya revisado a alguien en los últimos seis meses.
```
```suggestion
Comprueba la revisión más reciente de la persona y evita a cualquiera que haya revisado en los últimos seis meses.
```

yabellini commented on softwarereview_editor.es.Rmd:
```diff
@@ -70,28 +70,17 @@ Cuando estas fuentes de información no sean suficientes,
+- Puedes tuitear sobre la búsqueda de revisores.
 
-Tus herramientas para buscar personas para la revisión en *Airtable* son
+##### Consejos para la búsqueda de revisores en Airtable {#tips-for-reviewer-search-in-airtable}
```
```suggestion
##### Consejos para la búsqueda en Airtable {#tips-for-reviewer-search-in-airtable}
```

yabellini reviewed (commented):
Changes related to gender marks, grammar, and removing Twitter mention.

yabellini commented:
There is some 

> I added my first attempt at using {babeldown} to generate a translation. I've attempted to only stage the section of the file that I changed. Babeldown translates the whole file and many sections are different when it does so. (Likely a change anywhere has implications everywhere in the model, and also Babeldown's model may change over time.)

I think @maelle have now a function to only change the diff. But I still didn't use it.  We shouldn't change all the files because we localize; we don't only translate, and the translations are not literal, which means we will change the text to sound ok and more natural in Spanish.

noamross commented:
Thanks @yabellini! I made one more tiny tweak that I also auto-translated: [`236db7b` (#783)](https://github.com/ropensci/dev_guide/pull/783/commits/236db7b28e8e06e6deae711ba21716e15d450470#diff-5ac476b33feda191b6542ca112683cadafb3883b6e88c3e02f1735207484f371R71)

yabellini commented on softwarereview_editor.es.Rmd:
```diff
@@ -68,30 +68,17 @@ Cuando estas fuentes de información no sean suficientes,
 - También puedes buscar personas con paquetes relacionados en [r-pkg.org](https://r-pkg.org/).
 - R-Ladies tiene un [directorio](https://rladies.org/directory/) en el que se especifican las aptitudes e intereses de las personas incluidas en la lista.
-- Puedes publicar la búsqueda en las redes sociales.
+- Puedes publicar una solicitud de revisores en los canales #general y/o #software-review del Slack de rOpenSci, o en las redes sociales.
```
```suggestion
- Puedes publicar la búsqueda en los canales #general y/o #software-review del Slack de rOpenSci, o en las redes sociales.
```

yabellini commented on softwarereview_editor.es.Rmd:
```diff
@@ -68,30 +68,17 @@ Cuando estas fuentes de información no sean suficientes,
-No introduzcas más información que su nombre de usuario de GitHub, su nombre y su dirección de correo electrónico.
-Pide a quienes que podrían potencialmente revisar que completen el [formulario de Airtable](https://ropensci.org/software-reviewer).
+Comprueba la revisión más reciente de la persona y evita a cualquiera que haya revisado en los últimos seis meses.
+Asimismo, comprueba si una persona que es nueva revisando ha indicado que `require_mentorship`.
```
```suggestion
Asimismo, comprueba si una persona que es nueva revisando ha indicado que requiere tutoría en el campo`require_mentorship`.
```

yabellini reviewed (approved):
Small changes.  Looks good. 😄

Pakillo reviewed (approved):
Looks good to me, thanks!
